scripts/build/tests.mjs

```diff
@@ -9,6 +8,7 @@ import { findUpFile, findUpRoot } from "./findUpDir.mjs";
 import cmdLineOptions from "./options.mjs";
 import { exec, ExecError } from "./utils.mjs";
 
+const rf = { recursive: true, force: true };

```

Comment:
Though this saves typing (kinda), I'm not totally sure I like this compared to just writing them inline, which this script already does for `fs.promises.mkdir`.